packages/app-accounts/src/Banner.tsx

```diff
@@ -0,0 +1,105 @@
+// Copyright 2017-2019 @polkadot/app-accounts authors & contributors
```

Comment:
Not sure wht this was added here? Looks like a merge gone wrong. This file is in master.